src/Polysemy.hs

```diff
@@ -132,11 +136,11 @@ module Polysemy
   , Inspector (..)
   ) where
 
-import Polysemy.Internal
-import Polysemy.Internal.Combinators
-import Polysemy.Internal.Forklift
-import Polysemy.Internal.Kind
-import Polysemy.Internal.TH.Effect
-import Polysemy.Internal.Tactics
-import Polysemy.Final
+import           Polysemy.Final
+import           Polysemy.Internal
+import           Polysemy.Internal.Combinators
+import           Polysemy.Internal.Forklift
+import           Polysemy.Internal.Kind
+import           Polysemy.Internal.Tactics
+import           Polysemy.Internal.TH.Effect
```

Comment:
I'd say the extra indentation isn't needed here; I doubt we'll ever use quantified modules in `Polysemy`. The reordering is good, though.

Reply:
Oops! My setup indents imports by default, I forgot to turn this off. I will revert this changes!